.github/workflows/ci.yml

```diff
@@ -7,9 +7,9 @@ on:
       - main
       - stable/*
       - release-*
-  pull_request: {}
-  merge_group: {}
-  workflow_dispatch: {}
+  pull_request: { }
+  merge_group: { }
+  workflow_dispatch: { }
```

Comment:
🔧 Please avoid applying auto-formatting to files like this (I assume via an editor that formats on save according to some internal criteria) which just introduces noise in the Git history by adding spaces and makes review harder.

Reply:
I undid a good number of auto-save changes.
I decided to keep the bigger ones against the POMs, because those optimize files are now actually organized better than before.

Sorry about it. I deactivated the auto-format option in intelliJ now